### DISCUSSION TITLE
An error is reported when running a test case

### DISCUSSION THREAD

**Discussion by Yang6852** [PKU-Alignment/safety-gymnasium#89] [Q&A]
Hi,
https://www.safety-gymnasium.com/en/latest/introduction/basic_usage.html
After installing the safety_gymnasium with the pip command, I tested the Initializing the environment code in the above page.After the loop ends, the screen disappears according to the settings. However, the following error message is reported in the terminal
![2023-11-09 20-16-56屏幕截图](https://github.com/PKU-Alignment/safety-gymnasium/assets/66681868/7e2db5a5-0895-4e1c-b252-e8d6d7297151)
After switching to another environment, the error is still reported as "object has no attribute".
I would like to know how to solve the above problem.
Thanks.

**Comment by muchvo** [comment]
This issue was identified in a previous version of Gymnasium. As far as we know, it shouldn't affect normal usage. The problem occurs at the end of the program when objects are being released. If you'd like to resolve this warning, you might consider updating to a newer version of Gymnasium. You can also try our v1.2.0. We plan to provide official support for the latest version of Gymnasium in our upcoming releases.

If this issue is impacting your specific use case, please let us know. For more information, you can refer to this [issue](https://github.com/Farama-Foundation/Gymnasium/issues/325) which discusses it in detail.

**Comment by Yang6852** [comment]
来信已收到
